ginza/tests/test_command_line.py

```diff
@@ -7,6 +7,7 @@
 
 import pytest
 
+import torch
```

Comment:
[memo]
`torch` is dependency of `ja-ginza-electra`, and the tests assume that ja-ginza-electra is installed in advance, so there is no problem to use torch.